client/src/components/Auth/Auth.js

```diff
@@ -1,21 +1,63 @@
 import React, { useState } from 'react';
 import Login from './Login';
 import Signup from './Signup';
+import axios from 'axios';
+import store from '../../store/index';
 
 const Auth = () => {
-  const [tab, setTab] = useState('login');
+  const [page, setPage] = useState('login');
 
-  const login = (username, email, password) => {
-    console.log(username, email, password);
+  const login = (username, password) => {
+    axios
+      .post('/api/users/login', { username, password })
+      .then((res) => {
+        if (res.data.success) {
+          store.dispatch({
+            type: 'login',
+            _id: res.data.user._id,
+            user: res.data.user,
+            token: res.data.token,
+          });
+          window.location = '/';
+        } else {
+          console.log(res);
+        }
+      })
+      .catch((err) => {
+        console.log(err);
+      });
   };
 
-  let page = tab === 'login' ? <Login submit={login} /> : <Signup />;
+  const signup = (username, password) => {
+    axios
+      .post('/api/users/register', { username, password })
+      .then((res) => {
+        if (res.data.success) {
+          setPage('login');
+        }
+        console.log(res);
+      })
+      .catch((err) => {
+        console.log(err);
+      });
+  };
+
+  const changePage = () => {
```

Comment:
So I did not do separate pages for the login and sign up, it is a single page with a Container that has two tabs, one with the log in form and the other with the registration form. We should meet and discuss how to connect this with the UI implementation I did.